src/main/webapp/script.js

```diff
@@ -1,35 +1,47 @@
 async function displayCodes() {
-    var formInput = document.getElementById('formInput').value;
-    const response = await fetch('/promo-code?formInput=' + formInput);
+    let formInput = document.getElementById('formInput').value;
+    const selected = document.querySelector('input[name="searchOption"]:checked');
+    let response; 
+    // TODO: Change name of URI's to appropriate servlet names.
+    if (selected.id === "channel") {
```

Comment:
Is the triple equals intended here? Or rather, will the type for selected.id change to something with the value of "channel" yet is not a string which makes the additional type check necessary? Otherwise I'd suggest a simple ==.

Reply:
It is intended. The type will not change, I was just trying to follow the [google style guide ](https://google.github.io/styleguide/jsguide.html#features-equality-checks)